content/20.treatments.md

```diff
@@ -86,6 +86,68 @@ A list of current studies and their results, using carefully the information req
+This trial found HCQ decreased both time to fever break and time to cough relief.
+However, this study also showed design and analysis weaknesses that disallow general interpretation of results.
+These weaknesses include: lack of blinding, lack of placebo, inappropriate choice of outcomes, lack of correction for multiple primary outcomes, lack of sufficient detail to understand analysis, drastic disparities between pre-registration and published protocol, and small sample size.
+Correction for multiple primary outcomes can be adjusted post-hoc by recalculating p-values, but all other issues are design and statistical problems that cannot be corrected for.
+Additionally, the issue of drastic disparities between pre-registration and published protocol may be suggestive of p-hacking.
+Again, conclusions cannot de generalized outside of the study, but the results support further investigation.
+
+A randomized trial from the Shanghai Public Health Clinical Center of 30 COVID-19 patients studied whether HCQ increased rates of virological clearance by respiratory pharyngeal swab on Day 7 post-treatment compared to standard care [how to cite? http://www.zjujournals.com/med/CN/10.3785/j.issn.1008-9292.2020.03.03].
+This trial was published in Chinese with an abstract also in English.
+Only the English abstract was read and interpreted.
+The trial found HCQ showed comparable outcomes to standard care with regard to virological clearance rate, time to virological clearance, and time to body temperature normalization.
+A known weakness is small sample size.
+This problem suggests the study is underpowered to detect potentially useful differences and precludes interpretation of results.
+Additionally, because only the abstract could be read, other design and analysis issues could be present.
+Conclusions should be interpreted very cautiously.
+However, these preliminary negative results do reiterate the need for further study.
+
+A case study treated 11 consecutive patients with HCQ + azithromycin using the same dosing regime reported by Gautret et al [@doi:10.1016/j.medmal.2020.03.006].
+One patient died, two were transferred to the ICU, and one developed a prolonged QT interval leading to discontinuation of HCQ + azithromycin.
+As in the Gautret et al study, virological clearance was assessed at Day 6 post-treatment by nasopharyngeal swabs.
+Of the ten living patients at the time, eight of the patients remained positive for SARS-CoV-2 RNA.
+Interpretation of conclusions are severely limited by lack of comparison group and small sample size.
+However, these results stand in contrast to claims by Gautret et al and illustrates the need for better and further investigation.
+
+There is also conversation of clinical trials showing HCQ is superior to control treatment in inhibiting the exacerbation of pneumonia.
```

Comment:
"conversation of clinical trials" sounds a little confusing to me as terminology. Is there a way this could be potentially reworded a bit?

Reply:
Is this better: "Primary literature, review articles, and media have reported HCQ is superior..."
Or just dropped that part and said: "A Letter to the Editor was published in BioScience Trends claiming numerous clinical trials showed HCQ is superior to control treatment in inhibiting the exacerbation of pneumonia."

I guess my goal was to explain why we would even be discussing this low-quality Letter that provides no data. To me, the main relevance is that people are citing/referring to it without checking the citations it makes!

Reply:
I like the second formulation of a "A Letter to the editor..."! Definitely agree it's worth discussing if people are citing/referring to it...

Reply:
I made edits based on your suggestion. Could you take a look?

Reply:
New version is much better!